### PR TITLE
Add 'To add GxIT privileges' section to the subdomains

### PR DESCRIPTION
**Pull request by sanjaysrikakulam**
Linked PR: https://github.com/usegalaxy-eu/infrastructure/pull/178

### CODE REVIEW

File: subdomains.md [hunk 1]
```diff
@@ -27,6 +27,15 @@ In the website repository:
 1. Add your domain to [this list](https://github.com/usegalaxy-eu/infrastructure/blob/main/dns.tf#L24) under `subdomain`, and increase the `count` parameter below by one.
 2. Make a PR with these changes.
 
+### To add GxIT privileges
+
+To allow or grant your domain the GxIT privileges, the following needs to be done,
+
+1. Create a PR, adding your domains IT wildcard to the [sn06.yml playbook](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/master/sn06.yml#L41) like done in [this PR](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/916)
```
Comment (kysrpex) [inline subdomains.md:34]:
```suggestion
1. Create a PR, adding your domains IT wildcard to the [sn06.yml playbook](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/5ecffc1153fefc8f4ff1e44855ebf66a73c0593d/sn06.yml#L34) like done in [this PR](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/916)
```

I suggest to use a permalink so when lines are added or removed the link still makes sense (feel free to use another line).

File: subdomains.md [hunk 2]
```diff
@@ -27,6 +27,15 @@ In the website repository:
...
+2. Create a PR with the CNAME records routing to `usegalaxy.eu` in the [dns.tf file](https://github.com/usegalaxy-eu/infrastructure/blob/main/dns.tf) by adding your domain to [this list](https://github.com/usegalaxy-eu/infrastructure/blob/main/dns.tf#L241) under `it-subdomain`, and increase the `count` parameter below by one. Refer [this PR](https://github.com/usegalaxy-eu/infrastructure/pull/178)
```
Comment (kysrpex) [inline subdomains.md:35]:
```suggestion
2. Create a PR with the CNAME records routing to `usegalaxy.eu` in the [dns.tf file](https://github.com/usegalaxy-eu/infrastructure/blob/0bd0f81bd9d5ada2ac382da415adea6a910adec1/dns.tf) by adding your domain to [this list](https://github.com/usegalaxy-eu/infrastructure/blob/0bd0f81bd9d5ada2ac382da415adea6a910adec1/dns.tf#L244-L270) under `it-subdomain`, and increase the [`count` parameter below](https://github.com/usegalaxy-eu/infrastructure/blob/0bd0f81bd9d5ada2ac382da415adea6a910adec1/dns.tf#L276) by one. Refer to [this PR](https://github.com/usegalaxy-eu/infrastructure/pull/178)
```

Same permalink story.

**Review by kysrpex — APPROVED** [review]
Thanks! You can also go ahead without applying the suggested changes.

File: subdomains.md [hunk 3]
```diff
@@ -27,6 +27,15 @@ In the website repository:
 1. Add your domain to [this list](https://github.com/usegalaxy-eu/infrastructure/blob/main/dns.tf#L24) under `subdomain`, and increase the `count` parameter below by one.
 2. Make a PR with these changes.
 
+### To add GxIT privileges
+
+To allow or grant your domain the GxIT privileges, the following needs to be done,
```
Comment (bgruening) [inline subdomains.md:32]:
```suggestion
To run Galaxy Interactive Tools (GxIT) in your subdomain, we need to generate wildcard certificates for you. To do so the following needs to be done:
```

**Review by bgruening — APPROVED** [review]

**Pull request merged**